src/components/ErrorView.js

```diff
@@ -0,0 +1,98 @@
+import { Fragment } from 'react'
+import { div, h, iframe, pre } from 'react-hyperscript-helpers'
+import Collapse from 'src/components/Collapse'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+
+
+class StackTraceView extends Component {
+  render() {
+    const { lines } = this.props
+
+    return h(Collapse, {
+      style: { marginTop: '0.5rem' },
+      title: 'Stack Trace',
+      defaultHidden: true
+    }, [
+      pre({ style: { overflowX: 'auto' } }, [
+        lines.map(({ className, methodName, fileName, lineNumber }, idx) => div({ key: idx }, [
```

Comment:
Do we want to standardize whether we use `Array.map` or `_.map`?

Reply:
I needed `Array.map` to get the index. Other than that I don't really care.